install/installer/pkg/components/ws-manager/configmap_test.go

```diff
@@ -133,3 +137,58 @@ func TestBuildWorkspaceTemplates(t *testing.T) {
 		})
 	}
 }
+
+func TestWorkspaceURLTemplates(t *testing.T) {
+	tests := []struct {
+		Name                             string
+		Domain                           string
+		InstallationShortname            string
+		ExpectedWorkspaceUrlTemplate     string
+		ExpectedWorkspacePortURLTemplate string
+	}{
+		{
+			Name:                             "With an installation shortname",
+			Domain:                           "example.com",
+			InstallationShortname:            "eu02",
+			ExpectedWorkspaceUrlTemplate:     "https://{{ .Prefix }}.ws-eu02.example.com",
+			ExpectedWorkspacePortURLTemplate: "https://{{ .WorkspacePort }}-{{ .Prefix }}.ws-eu02.example.com",
+		},
+		{
+			Name:                             "Without an installation shortname",
+			Domain:                           "example.com",
+			InstallationShortname:            "",
+			ExpectedWorkspaceUrlTemplate:     "https://{{ .Prefix }}.ws.example.com",
+			ExpectedWorkspacePortURLTemplate: "https://{{ .WorkspacePort }}-{{ .Prefix }}.ws.example.com",
```

Comment:
:+1:

Reply:
@andrew-farries We need a test case "with 'default' as installation shortname" to cover all existing self-hosted installations out there I think. That will lead us to [the comment above](https://github.com/gitpod-io/gitpod/pull/10152/files#r883577546) I think.